src/Ledger/Utxo.lagda

```diff
@@ -372,10 +370,12 @@ data _⊢_⇀⦇_,UTXO⦈_ where
         open UTxOState s
     in
     ∙ txins ≢ ∅                              ∙ txins ⊆ dom utxo
-    ∙ inInterval slot txvldt                 ∙ feesOK pp tx utxo ≡ true
-    ∙ consumed pp s txb ≡ produced pp s txb  ∙ coin mint ≡ 0
-    ∙ txsize ≤ maxTxSize pp
+    ∙ refInputs ⊆ dom utxo                   ∙ inInterval slot txvldt
+    ∙ feesOK pp tx utxo ≡ true               ∙ consumed pp s txb ≡ produced pp s txb
+    ∙ coin mint ≡ 0                          ∙ txsize ≤ maxTxSize pp
 
+    ∙ ∀[ (_ , txout) ∈ txouts .proj₁ ]
+        inject (((serSize (getValue txout)) + 160) * coinsPerUTxOWord pp) ≤ᵗ getValue txout
```

Comment:
This is unnecessary, the next condition is a more recent version of this condition.